const content = {
  title: 'Project Outline',
  sections: [
    {
      subtitle: 'Pneumococcal Disease',
      content: [
        {
          text: '*Streptococcus pneumoniae* (the pneumococcus) is the leading vaccine preventable cause of pneumonia, septicaemia and meningitis in young children. In 2015, it was estimated to be responsible for about 9 million cases of disease and over 317,000 deaths in children under 5 with the main burden in low- and middle- income countries ([Wahl 2018](https://www.thelancet.com/journals/langlo/article/PIIS2214-109X(18)30247-X/fulltext)).',
        },
        {
          text: '![Image of Streptococcus pneumoniae](streptococcus.jpg)'
        },
        {
          text: 'Pneumococcal conjugate vaccines (PCVs) target the capsule surrounding pneumococcal cells has proven to be very effective in reducing pneumococcal disease since its advent in 2000. However, there are at least [100 different forms of capsule](../serotypes), or serotypes, based on antigen-antibody reaction. The current PCVs are only able to target some serotypes and the population overall is able to evolve to evade the vaccine. This creates an arms race between vaccine and pneumococcus.'
        },
        {
          text: '![Current PCV formulations](PCV_formulation.png)'
        },
      ]
    },
    {
      subtitle: 'GPS',
      content: [
        {
          text: 'The Global Pneumococcal Sequencing (GPS) project, was set out to create a global picture of the pneumococcal evolution during the vaccine introductions using whole-genome sequencing. The information has enabled us to better understand how the pneumococcus makes its capsule and makes its escape, and improve the design of new vaccines.'
        },
        {
          text: "The GPS project was conceptualised by Prof Stephen Bentley at the Wellcome Sanger Institute and Prof Keith Klugman in Emory university over a pint in The Queen's Arms in Kensington, London in 2009. They discuss the feasibility of carrying out a genomic surveillance programme that would sequence the genomes of over 20,000 pneumococcal bacteria collected from around the world. Initially the focus was to better understand the frequency of antibiotic resistance and to discover new resistance mechanisms but they were soon persuaded by the Bill & Melinda Gates Foundation that the primary focus should be on understanding vaccine escape."
        },
        {
          text: 'To achieve such a large-scale project, they needed help from other world-leading scientists. Dr Lesley McGee from the US Centers for Disease Control and Prevention, an expert in pneumococcal microbiology and vastly experienced in large scale international studies, was brought on board as one of the project leaders. They also established strategic collaborations with other experts: Dr Anne von Gottberg from the National Institute Communicable Diseases, South Africa; Prof Martin Antonio from the Medical Research Unit The Gambia at the London School of Hygiene & Tropical Medicine; and Prof Dean Everett from Malawi-Liverpool-Wellcome Trust Clinical Research Programme.'
        },
        {
          text: 'And when Keith took up a new role as Head of the Pneumonia Program at the Bill & Melinda Gates Foundation, Prof Robert Breiman from Emory University took over the reigns as Principal Investigator. With the financial support from the Gates Foundation, the Sanger Institute and the US Centers for Disease Control and Prevention, the Global Pneumococcal Sequencing (GPS) project was established in 2011.'
        },
        {
          text: 'By the end of 2019, the GPS project sequenced >26,000 pneumococcal isolates from >50 countries, representing a global collaboration of almost 100 scientists from around the world. Like previous studies, we have seen that after the introduction of vaccines, pneumococci with vaccine serotypes were replaced by those with non-vaccine serotypes. But with large-scale genome data, we have gone beyond that, looking for the first time at all the strains circulating globally.'
        },
        {
          text: 'The GPS project has made a major breakthrough in defining pneumococcal strains in high-resolution by taking genome-wide DNA variations into account ([Gladstone and Lo et al 2019](https://www.thelancet.com/article/S2352-3964(19)30259-2/fulltext)). This robust typing method is coupled with a coherent and dynamic naming system ([GPSC](https://www.pneumogen.net/gps/GPSC_lineages.html)) so as to enable joint efforts in tracking pneumococcal lineages across geographical regions and over time, especially before and after the roll-out of PCVs.'
        },
        {
          text: 'Using this typing scheme, we found a subset of strains were globally disseminated, many of which were associated with antibiotic resistance and composed of vaccine- and non-vaccine serotypes. Our recent work, published in the Lancet Infectious Diseases, has shown that these globally-spreading strains played an important role in the arms race by driving the expansion of pneumococcal bacteria with non-vaccine targets ([Lo and Gladstone et al 2019](https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(19)30297-X/fulltext)).'
        },
        {
          text: 'The DNA data also allowed us to unlock the genetic diversity of the capsule and discovered nine potential new serotypes ([van Tonder et al 2019](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000274)) and shed light on previously unrecognised antimicrobial resistance in pneumococcus ([Lo et al 2019](https://academic.oup.com/jac/article/75/3/512/5650366)). For more exciting findings, please browse our [publications page](../publications).'
        },
      ]
    },
    {
      subtitle: 'Open Data and Findings',
      content: [
        {
          text: 'The GPS project generates a rich source of genome and epidemiological data for tracking pneumococcal transmission and evolution on both national and international levels. The genome data are deposited in [ENA](https://www.ebi.ac.uk/ena/data/view/PRJEB3084). High-level analyses are interactively presented by [countries](../countries) and by [strains](../strains) using [Microreact](https://microreact.org/) and [Phandango](https://jameshadfield.github.io/phandango/#/GPS). Illustrations of how to use these resources are summarised in a recent publication ([Gladstone et al 2020](https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000357?originator=authorOffprint&identity=176964&timestamp=20210430151417&signature=3ebadc206d0b1fda9098a00fc65d31ec)) and videos on the [Resources pages](../overview). '
        },
      ]
    },
    {
      subtitle: "What's Next?",
      content: [
        {
          text: 'In 2020, the GPS project is marching into the next chapter, GPS2. The outputs of GPS highlight the needs on'
        },
        {
          text: '1. **Increasing geographical coverage** in countries, especially those with high disease burden, such as India, Nigeria, DRC Congo and Pakistan, together these four countries accounted for approximately 50% of global pneumococcal disease burden in 2015.'
        },
        {
          text: '2. **Supporting local data generation and analysis** to achieve a working system for long-term and sustainable genomic surveillance in LMICs by providing capacity building and [bioinformatics training](https://training.bactgen.sanger.ac.uk/).'
        },
        {
          text: '3. **Engaging with policy makers** to communicate the value of the data and the importance of study findings for decision-making in pneumococcal disease prevention.'
        },
        {
          text: 'The success of the GPS project has also opened up opportunities to establish strategic partnership with'
        },
        {
          text: '1. [SIREVA](https://redsirevanetwork.com/) to establish a framework for integrating whole-genome sequencing in the ongoing surveillance in Latin America.'
        },
        {
          text: "2. [Professor Moon Nahm's lab](https://www.vaccine.uab.edu/) at the University of Alabama to characterise capsule structure of the nine potential new serotypes."
        },
        {
          text: "3. [NHMRC Centre of Research Excellence for Pneumococcal Disease Control in the Asia-Pacific (CRE-PDC)](https://www.mcri.edu.au/research/strategic-collaborations/centres/centre-pneumococcal-disease-control-asia-pacific) at the Murdoch Children's Research Institute to understand the sustainability of national immunisation PCV programs in the Asia-Pacific region."
        },
        {
          text: ' Last but not least, we thank the [Bill & Melinda Gates Foundation](https://www.gatesfoundation.org/) to fund this project.'
        },
      ]
    },
    {
      subtitle: "Related Links",
      content: [
        {
          text: '- [The Story of GPS](https://sangerinstitute.blog/2019/06/12/tracking-a-deadly-shapeshifter)'
        },
        {
          text: '- [GPS Headline Findings](https://www.sanger.ac.uk/news_item/pneumonia-mapped-largest-genomic-survey-any-disease-causing-bacterium/)'
        },
        {
          text: '- [VIEW-hub: Reports on disease burden, vaccine access and coverage](https://view-hub.org/)'
        },
        {
          text: '- [Gavi: the Vaccine Alliance](https://www.gavi.org/)'
        },
      ]
    },
  ]
}

export default content